repo/packages/M/mysql/0/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "packagingVersion": "2.0",
+  "name": "mysql",
+  "version": "5.7.12",
+  "scm": "https://github.com/mysql/mysql-server.git",
+  "maintainer": "support@mesosphere.io",
+  "website": "https://mysql-ci.org",
+  "description": "MySQL is the world's most popular open source database. With its proven performance, reliability and ease-of-use, MySQL has become the leading database choice for web-based applications, covering the entire range from personal projects and websites, via e-commerce and information services, all the way to high profile web properties including Facebook, Twitter, YouTube, Yahoo! and many more.",
+  "tags": ["database", "mysql", "sql"],
+  "preInstallNotes": "WARNING: MySQL on DCOS is currently in ALPHA. There may be bugs, incomplete\nfeatures, incorrect documentation, or other discrepancies.\n\nIf you didn't provide a value for `host_volume` in the CLI,\nYOUR DATA WILL NOT BE SAVED IN ANY WAY.\n",
```

Comment:
Please put how many resources are required by default into the `preInstallNotes`.